Assets/Scripts/PlayerCharacter.cs

```diff
@@ -0,0 +1,170 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.SceneManagement;
+
+public class PlayerCharacter : MonoBehaviour
+{
+    [SerializeField]
```

Comment:
Combine all Serialized fields into a Region.

Done by starting an empty line with #Region and ending that cluster with #endregion. Contains what's inside like an if statement.